### PR TITLE
Add FatwaDwiHidayat to the list of interns

### PR DESCRIPTION
**Pull request by FatwaDwiHidayat**
Preview URL is https://raw.githack.com/FatwaDwiHidayat/open-source/feature/add-FatwaDwiHidayat/en_ID/#!index.md

### CODE REVIEW

File: en_ID/pages/profiles/FatwaDwiHidayat.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+#FatwaDwiHidayat
+
+Hi I'm [FatwaDwiHidayat](http://github.com/FatwaDwiHidayat). I like software engineering because it is fun! And my favorite food is burger.
```
Comment (empeje) [inline en_ID/pages/profiles/FatwaDwiHidayat.md]: This is too simple, try to be more creative ya

Comment (empeje) [reply en_ID/pages/profiles/FatwaDwiHidayat.md]: You can put images, videos, or any additional things that describe you!

**Comment by empeje** [comment]
Thank you for making your first PR, please resolve any conflicts, and please complete the entire first steps.

File: en_ID/pages/profiles/FatwaDwiHidayat.md (hunk 1 above)
Comment (FatwaDwiHidayat) [reply en_ID/pages/profiles/FatwaDwiHidayat.md]: I had doing my revision. Please review ya

**Comment by empeje** [comment]
Thank you for your update @FatwaDwiHidayat, but the Youtube Image in your profile page seems broken 😢 

Also please resolve conflicts.

This is material on how to resolve Git conflicts https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/resolving-a-merge-conflict-on-github

**Comment by empeje** [comment]
Close in lieu of https://github.com/kulkultech/open-source/pulls?q=is%3Apr+is%3Aclosed

**Pull request closed without merging**